scripts/src/main/resources/scripts/command/ide

```diff
@@ -42,7 +42,13 @@ function doUpdateSettings() {
         echo "Please contact the technical lead of your project to get the SETTINGS_URL for your project."
         echo "In case you just want to test devonfw-ide you may simply hit return to install default settings."
         echo
-        read -r -p "Settings URL: " answer
+        if [ "${DEFAULT_SETTING_URL}" == "default" ]
+        then
+          echo  "Settings URL:"  "${DEFAULT_SETTING_URL}"
+          SETTINGS_URL="-"
+        else
+          read -r -p "Settings URL: " answer
+        fi
```

Comment:
why do you need this change after all?
Instead of supplying "default" you can already supply "-" and you do not need this change at all.